lib/matplotlib/axes/_axes.py

```diff
@@ -5037,23 +5039,25 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
 
         Parameters
         ----------
-        X : array_like, shape (n, m) or (n, m, 3) or (n, m, 4)
+        X : array_like, shape (n, m) or (n, m, 3) or (n, m, 4) or (2, n, m)
             Display the image in `X` to current axes.  `X` may be an
             array or a PIL image. If `X` is an array, it
             can have the following shapes and types:
 
-            - MxN -- values to be mapped (float or int)
+            - MxN -- univariate values to be mapped (float or int)
             - MxNx3 -- RGB (float or uint8)
             - MxNx4 -- RGBA (float or uint8)
+            - 2xMxN -- bivariate values to be mapped (float or int)
 
             The value for each component of MxNx3 and MxNx4 float arrays
-            should be in the range 0.0 to 1.0. MxN arrays are mapped
+            should be in the range 0.0 to 1.0. MxN and 2xMxN arrays are mapped
             to colors based on the `norm` (mapping scalar to scalar)
             and the `cmap` (mapping the normed scalar to a color).
 
-        cmap : `~matplotlib.colors.Colormap`, optional, default: None
+        cmap : `~matplotlib.colors.Colormap`,
```

Comment:
This is the one exception to the `()` over `\` rule so that numpydoc correctly grabs this as a single lin.

Reply:
These might be the source of the docs failures?

Reply:
Added `\` but still failing.

Reply:
It is probably fastest to build the docs locally.  That also lets you get access to the full traceback.

```bash
pip install -r doc-requirements.txt
cd docs
python make.py html
```

*should* work.

Reply:
```
Generating gallery...
Generating gallery for gallery/api [100%] watermark_text.py                                                                                                                  
Generating gallery for gallery/pyplots [ 78%] whats_new_1_subplot3d.py                                                                                                       
Warning, treated as error:
/home/harshit/Documents/matplotlib/examples/pyplots/whats_new_1_subplot3d.py failed to execute correctly:Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/sphinx_gallery/gen_rst.py", line 450, in execute_code_block
    exec(code_block, example_globals)
  File "<string>", line 9, in <module>
  File "/home/harshit/Documents/matplotlib/lib/matplotlib/figure.py", line 1070, in add_subplot
    a = subplot_class_factory(projection_class)(self, *args, **kwargs)
  File "/home/harshit/Documents/matplotlib/lib/matplotlib/axes/_subplots.py", line 73, in __init__
    self._axes_class.__init__(self, fig, self.figbox, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/mpl_toolkits/mplot3d/axes3d.py", line 121, in __init__
    self.axesPatch.set_linewidth(0)
AttributeError: 'function' object has no attribute 'set_linewidth'

Building HTML failed.
```
This is the error I am getting.

Reply:
Well but in `test_mplot3d.py` most of the test on my system fails.